cclient/request/store_transactions.h

```diff
@@ -16,13 +16,13 @@ extern "C" {
 
 typedef struct store_transactions_req_s {
   // List of raw transactions data to be stored
-  flex_hash_array_t* trytes;
+  hash8019_queue_t trytes;
```

Comment:
Order does not matter as well, maybe use a stack ?

Reply:
sure, thanks :-)